index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header-section">
+        <a href="" class="logo"> My logo </a>
+<nav class="bttn">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+</nav>
+    </header>
+    <section  >
+        <img class="background-image"src="i-bttn/background-shapes/header-shapes mobile.png" alt="">
+        <p class="text"> <h1>I’m Trevor <br> Glad to see you!</h1></p> 
```

Comment:
- [ ] Good job adding this `h1` tag, but it is not a good practice to wrap it into a `p` tag. Kindly make sure you change that :+1: